appengine/pond/js/avatar.js

```diff
@@ -34,22 +34,46 @@ goog.require('BlocklyInterface');
  * Class for a avatar.
  * @param {string} name Avatar's name.
  * @param {string|!Function} code Avatar's code, or generator.
- * @param {!Blockly.utils.Coordinate} startLoc Start location.
  * @param {?number} startDamage Initial damage to avatar (0-100, default 0).
- * @param {!Pond.Battle} battle The battle featuring the avatar.
+ * @param {?Blockly.utils.Coordinate} opt_startLoc Start location.
```

Comment:
optional paramter is defined in jsdoc with `{<type>= } `and optionally can use `[name=<default value>]` to explain default value.